src/forms/services/form-submission.service.ts

```
@@ -1593,11 +1918,42 @@
+              Array.isArray(fieldsObj?.result) &&
+              fieldsObj.result[0]?.schema?.properties
+            ) {
+              // Structure: { result: [{ schema: { properties: {...} } }] }
+              schema = fieldsObj.result[0].schema.properties;
+            } else if (fieldsObj?.schema?.properties) {
+              // Structure: { schema: { properties: {...} } }
+              schema = fieldsObj.schema.properties;
+            } else if (fieldsObj?.properties) {
+              // Structure: { properties: {...} }
+              schema = fieldsObj.properties;
+            } else if (typeof fieldsObj === 'object' && fieldsObj !== null) {
```

ESLint

[error] Replace `'object'` with `"object"`

(prettier/prettier)